src/mrpro/data/traj_calculators/_KTrajectoryCalculator.py

```diff
@@ -78,8 +78,8 @@ class DummyTrajectory(KTrajectoryCalculator):
     Shape will fit to all data. Only used as dummy for testing.
     """
 
-    def __call__(self, header: KHeader) -> KTrajectory:
-        """Calculate dummy trajectory for given KHeader."""
-        kx = torch.zeros(1, 1, 1, header.encoding_limits.k0.length)
+    def __call__(self, header: KHeader) -> KTrajectory:  # noqa: ARG002
```

Comment:
why do we keep the header argument if we don't need it anymore?

Reply:
because otherwise the signature of the function does not match anymore